frontend/src/components/entity/AvailableEntity.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+
+import LoaderWrapper from 'src/components/LoaderWrapper';
+import { TypeEnum } from 'src/services/openapi';
+import { ActionList } from 'src/utils/types';
+import { idFromUid } from 'src/utils/video';
+
+/**
+ * Return an <EntityNotAvailable> if the entity doesn't seem to be available
```

Comment:
Let's rework this comment. The mentionned `<EntityNotAvailable>` component doesn't exist anymore.